site/src/hooks-doc/useTernaryDarkMode/useTernaryDarkMode.demo.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import { useTernaryDarkMode } from 'usehooks-ts'
+
+export default function Component() {
+  const { isDarkMode, ternaryDarkMode, toggleTernaryDarkMode } =
+    useTernaryDarkMode()
+
+  return (
+    <div>
+      <p>Current theme: {isDarkMode ? 'dark' : 'light'}</p>
+      <p>ternaryMode: {ternaryDarkMode}</p>
+      <button onClick={toggleTernaryDarkMode}>
+        Toggle from {ternaryDarkMode}
+      </button>
+      <p>
+        enum strict equal test:{' '}
+        {ternaryDarkMode === 'system' ? 'using System' : 'not using System'}
+      </p>
```

Comment:
Add an example with the new method `setTernaryDarkMode` like a simple `<button>`

Reply:
added on `da06b5c`